src/store/sagas/pool.ts

```diff
@@ -31,13 +31,13 @@ export function* fetchPoolsData(action: PayloadAction<Pair[]>) {
     }
 
     yield* put(actions.setPools(pools))
-    for (let i = 0; i < PAIRS[networkType].length; i++) {
-      const ticksArray = yield* call(
-        [marketProgram, marketProgram.getAllTicks],
-        PAIRS[networkType][i]
-      )
-      yield* put(actions.setTicks({ index: i, tickStructure: ticksArray }))
-    }
+    // for (let i = 0; i < PAIRS[networkType].length; i++) {
```

Comment:
remove commented code unless it will be used later

Reply:
done